src/operations/expansions/expandPath2.js

```diff
@@ -0,0 +1,71 @@
+const { area } = require('../../math/utils')
+
+const { vec2 } = require('../../math')
+
+const { geom2, path2 } = require('../../geometry')
+
+const offsetFromPoints = require('./offsetFromPoints')
+
+/**
+ * Expand the given geometry (path2) using the given options (if any).
+ * @param {Object} options - options for expand
+ * @param {Number} [options.delta=1] - delta (+) of expansion
+ * @param {Integer} [options.segments=0] - number of segments when creating rounded corners, or zero for chamfer
+ * @param {path2} geometry - the geometry to expand
+ * @returns {geom2} expanded geometry
+ */
+const expandPath2 = (options, geometry) => {
```

Comment:
very awesome ! (despite my brain still breaking when thinking about what an expansion of a non closed path means ?)

Reply:
I suggest we play with this once integrated into the viewer. It will be far easier to perceive, and changes may be needed.

Reply:
@kaosat-dev do you remember this little piece of wonder?
```
    // Expand the path to a CAG
    // This traces the path with a circle with radius pathradius
  expandToCAG: function (pathradius, resolution)
```
So, expanding a path is not new. BUT, a whole lot more sensical.

Reply:
Resolving. We can address this later if necessary.